// guptaabhi/service.js
// const tbl1 = require("./module/user");
// let tble2=require("./module/user3.0");
// let tble1=require("./module/user2.0");


// exports.ser_inserts = async (req, res) => {
//     let a;
//     let b = req.body.pa; 
//     let c = req.body.na;
//     let d = req.body.mn;
//     let e = req.body.ag;
// console.log("data aa gya kya:="+b+" "+c+" "+d+" "+e);
//     let hp=await bc.hash(b,10);
//     console.log(hp);
        
        
    
//         let data = await tbl1.findOne().sort({ account: -1 });

//         if (data) {
//             a = data.account + 1;
//         } else {
//             a = 1;
//         }
//         console.log(`Assigned account number: ${a}`);

//         let rec =await new tbl1({ account: a, password: hp, name: c, mobile: d, age: e });
//         await rec.save();

//         res.render("created");
        
        
//     };




// exports.ser_display=async(req,rep)=>
// { 
//     let a=req.query.an;
//     let b=req.query.pa;
//     let data2=await tbl1.findOne({account:a});
//     let v=await bc.compare(b,data2.password);
//     console.log(v);
//     if(v==true){ 
//         console.log("hello");
//         b=data2.password;
//         let data=await tbl1.findOne({account:a,password:b});
//         console.log(data);
//         if(data){
//             rep.cookie("acn",data.account);
//          rep.render("wel",{result:data.account});
//      }
//          else{ 
//                 rep.render("newc");
//          }
    
//     }
//     else{
//         rep.render("error");
//     }

    
//     }
    
//     exports.ser_dep=async(req,rep)=>
//     {
//         let fa=req.cookies.acn;
//         console.log(fa);
//         let ga=req.query.da;
//         let dp="deposit";
//         console.log(dp);
//         let data=await tble1.findOne({accounts:fa});
//         console.log(data);
//         console.log(fa);
//         if(data){
//             console.log("hello1");
//             await tble1.findOne({accounts:fa});
//             await tble1.updateOne({accounts:fa},{$inc:{amounts:+ga}});
//             let rec =await new tble2({ accounts:fa,amounts:ga,type:dp,date:Date});
//         await rec.save();

//             rep.render("deposuccess");

//         }
//         else{
//             console.log("hello2");
//             let abhi=new tble1({accounts:fa,amounts:ga});
//             abhi.save();
//             rep.render("deposuccess");
//         }
    
    
//         console.log(req.cookies.acn);

        
//     }
    

//     exports.ser_wid=async(req,rep)=>
//         {
//             let fa=req.cookies.acn;
//             let wa=req.query.wa;
//             let wd="Widthraw";
//             console.log(wd);
            
//         await tble1.findOne({accounts:fa});
//         await tble1.updateOne({accounts:fa},{$inc:{amounts:-wa}});
//         let rec =await new tble2({ accounts:fa,amounts:wa,type:wd});
//         await rec.save();
//         rep.render("widrawsuccess");
    
    
//         }

//         exports.ser_check=async(req,rep)=>
//         {
//             let fa=req.cookies.acn;
//             let data=await tble1.findOne({accounts: fa});
//             let balance=data.amounts;
//             rep.render("balance",{ balance: balance});
//         }
//         exports.ser_tran=async(req,rep)=>
//         {
//             let fa=req.cookies.acn;
//             let data=await tble2.find({accounts: fa},{_id:0,});
//             let transaction1=data;
//             rep.render("transaction",{transaction: transaction1});
//         }
